Add books to favourites catalog

kgogola opened the pull request:
In Book component I included button which allow to add some books to favourites catalog. User can add book just once, so it can't be a couple of same books in this catalog. 
I also included simple layout to watching results, but it should be replaced with layout from my previous task.

kgogola committed:
Adding books to favourites catalog

gckprs commented on components/Book.vue:
```diff
@@ -1,17 +1,47 @@
+  },
+  props: ['title', 'author', 'description', 'img', 'id', 'nrFav'],
+  methods: {
+    addFavourite () {
```
This way you just unnecessarily duplicate data. I think the logic would be much easier / more effective if every book simply had predicate isFavourite property. Then adding / removing would only take one operation. In addition to that you could easily _.filter() books to get favourite ones.
Someone correct me if I'm wrong, please.

kwojcik-blockether replied:
Agree with you

k911 commented:
@kgogola 
Commit message:
`Adding books to favourites catalog` -> `Add books to favourites catalog`

```bash
$ git commit --amend "Add books to favourites catalog"
$ git push origin PUT_YOU_BRANCH_NAME_HERE --force
```

Commit messages should always be written in imperative mode

k911 commented:
@kgogola You have to run script I've provided you in your console in your project directory

kgogola committed:
Add existing file

kgogola committed:
Add books to favourites catalog

kgogola commented:
@k911 I' ve got some troubles with editing commit. I get such error:
"error: pathspec 'Add books to favourites catalog' did not match any file(s) known to git."
I was trying edit it by other devices as well but also without succes, end that's why I created new commit.
Could you explain me what might be incorectly with it? I was using Git Bash

k911 commented:
I think that you've really messed up here 😄 

You you can do interactive rebase:

```bash
$ git rebase -i HEAD~3
```

and accordingly 'reword' fist commit, to proper name which is 'Add books to favourites catalog'
and fixup commits bellow like here: (Another project, but you should see almost the same view)

![screen shot 2018-04-19 at 17 22 19](https://user-images.githubusercontent.com/10072889/39001293-3ea73aa2-43f6-11e8-82fd-ad875995db8f.png)

LaYatiX reviewed (changes requested):
Can't accept it for now. If you have any question please contact me on slack

kgogola committed:
Add books to favourites catalog (Geecek's idea)

Now data aren't copied. Button change ''isFav'' property and it's allows to display favourites books in appropriate catalog.

kgogola commented:
Sorry guyz for time lag, I had some issues :/ I hope the new update will be correct. Unfortunately still I can't change this name of commit, I don't really now @k911 what did I break there, now I get such notice:
"Cannot rebase: Your index contains uncommitted changes.
Please commit or stash them."

In git status I have certain new file. I think here is the problem. Reset command not working and cached as well.
